src/helpers.ts

```diff
@@ -473,26 +475,37 @@ export function getShuffleItems(
     userId: string,
     item: BaseItemDto
 ): Promise<BaseItemDtoQueryResult> {
-    const query: ItemQuery = {
-        Fields: requiredItemFields,
-        Filters: 'IsNotFolder',
-        Limit: 50,
-        Recursive: true,
-        SortBy: 'Random',
-        UserId: userId
+    const query: ItemsApiGetItemsRequest = {
+        fields: requiredItemFields,
+        filters: ['IsNotFolder'],
+        limit: 50,
+        recursive: true,
+        sortBy: ['Random'],
+        userId
```

Comment:
User id doesn't have to be provided as it is inferred from the access token since Jellyfin 10.9